cmd/linux.go

```diff
@@ -120,6 +120,9 @@ func parseOsRelease(osRelease string) *DistroInfo {
 	return result
 }
 
+// GetPkgInstalled is all functions that use local programs to see if a package is installed
+type GetPkgInstalled func(string) (bool, error)
```

Comment:
may I suggest changing this to `CheckPkgInstalled`? It is a bit confusing as it seems to me that it'll get a pkg installed.